openedx/core/djangoapps/schedules/admin.py

```diff
@@ -11,11 +11,45 @@ class ScheduleExperienceAdminInline(admin.StackedInline):
 
 @admin.register(models.Schedule)
 class ScheduleAdmin(admin.ModelAdmin):
-    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline')
+    list_display = ('username', 'course_id', 'active', 'start', 'upgrade_deadline', 'experience_display')
+    list_filter = ('experience__experience_type', 'active')
     raw_id_fields = ('enrollment',)
     readonly_fields = ('modified',)
-    search_fields = ('enrollment__user__username', 'enrollment__course_id',)
+    search_fields = ('enrollment__user__username', 'enrollment__course__id',)
     inlines = (ScheduleExperienceAdminInline,)
+    actions = ['deactivate_schedules', 'activate_schedules', 'set_experience_to_default', 'set_experience_to_course_updates']
+
+    def deactivate_schedules(self, request, queryset):
+        rows_updated = queryset.update(active=False)
+        self.message_user(request, "{} schedule(s) were deactivated".format(rows_updated))
+    deactivate_schedules.short_description = "Deactivate selected schedules"
+
+    def activate_schedules(self, request, queryset):
+        rows_updated = queryset.update(active=True)
+        self.message_user(request, "{} schedule(s) were activated".format(rows_updated))
+    activate_schedules.short_description = "Activate selected schedules"
+
+    def set_experience_to_default(self, request, queryset):
```

Comment:
Question: Although this is fine for now, is there a relatively easy way to have the admin specify the experience-type (by number) and have one admin control for updating to the specified experience type?  Otherwise, we'll need to add a new admin control for each experience type we add.

Reply:
We could have the action take you to an interstitial page, or we could use a for-loop to generate action functions.

Reply:
fixed